src/UI/Handlebars/Helpers/Html.js

```diff
@@ -14,15 +14,24 @@ window.NzbDrone.imageError = function(img) {
 };
 
 Handlebars.registerHelper('defaultImg', function(src, size) {
+    var endOfPath = /\.jpg($|\?)/g;
+    var errorAttr = 'onerror="window.NzbDrone.imageError(this);"';
+    var srcsetAttr = '';
+    var oneX, twoX;
+
     if (!src) {
-        return new Handlebars.SafeString('onerror="window.NzbDrone.imageError(this);"');
+        return new Handlebars.SafeString(errorAttr);
     }
 
     if (size) {
-        src = src.replace(/\.jpg($|\?)/g, '-' + size + '.jpg$1');
+        oneX = src.replace(endOfPath, '-' + size + '.jpg$1');
+        twoX = src.replace(endOfPath, '-' + size * 2 + '.jpg$1');
+        srcsetAttr = 'srcset="{0} 1x, {1} 2x"'.format(oneX, twoX);
```

Comment:
I'd love to use template strings here instead of a prototype-extending format method. E.g 
```javascript
`srcset="${oneX} 1x, ${twoX} 2x"`
```
Thoughts about setting up [babel](https://babeljs.io/) for this and other ES6 features?